File: MODUL.2/PRAK204_2310817120010_NurHikmah.py
print('Input')
bilangan = input()

nilai = bilangan.split()

r, t = map(int, nilai)

phi = 22 / 7
V = phi * (r**2) * t
L = 2 * phi * r * (r + t)
K =  2 * phi * r

print('\nOutput')
print(f'Volume   = {V:.2f}')
print(f'Luas     = {L:.2f}')
print(f'Keliling = {K:.2f}')

File: MODUL.2/PRAK204_2310817120010_NurHikmah.c
#include <stdio.h>
#include <math.h>

#define phi 3.142857

int main()
{
    float r, t, V, L, K;

    printf("Input\n");
    scanf("%f %f", &r, &t);

    V = phi * r * r * t;
    L = 2 * phi * r * (r + t);
    K = 2 * phi * r;

    printf("\nOutput\nVolume    = %.2f\n", V);
    printf("Luas      = %.2f\n", L);
    printf("Keliling  = %.2f\n", K);

    return 0;
}
